### DISCUSSION TITLE
Default z-index design tokens

### DISCUSSION THREAD

**Discussion by henrieri** [twilio-labs/paste#901] [Q&A] [answered]
Hi Team!

Default theme seems to have z-index tokens starting from 0 and ending with 90, while for Legacy it's from 2000 to 2090.
Is this intentional? 

See: https://paste.twilio.design/tokens

Right now when I'm trying to use default theme in Console, modal's dark background won't cover console nav and header.
![image](https://user-images.githubusercontent.com/8214414/97902501-64bdab00-1d46-11eb-82ad-53bcb4ed350e.png)

Thank you in advance.

**Comment by richbachman** [comment, marked as answer]
Hi @henrieri 

Yes that token value difference is intentional. The default theme is intended to be used with the new Console navigation and header, so the lower number values in the default theme will work.

@hharnisc do you know if there is a way for downstream teams to know they are using the new navigation? If so, they could switch themes based on some sort of flag.

**Comment by henrieri** [comment]
@richbachman thank you for a quick answer. 

If there's an account flag for it, it would be quite easy to check. 
How do you test with the new nav currently or how could I see the nav?

**Reply by TheSisb** [reply]
@henrieri Sorry for the slow reply. You should be able to get help with that in #help-console-zen on Slack.
